openedx_events/learning/signals.py

```diff
@@ -196,3 +197,39 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_THREAD_CREATED
+# .. event_description: Emitted when a new thread is created in a discussion
+# .. event_data: DiscussionThreadData
+# Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
+FORUM_THREAD_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.thread.created.v1",
+    data={
+        "thread": DiscussionThreadData,
+    }
+)
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_RESPONSE_CREATED
+# .. event_description: Emitted when a new response is added to a thread
```

Comment:
What's the difference between a response and a comment?

Reply:
Threads are responded to, while responses are commented.

![image](https://github.com/openedx/openedx-events/assets/33465240/7dfe0eab-66ac-4fb0-9eb1-28c36bbeb7e6)

Reply:
The name is: `FORUM_THREAD_RESPONSE_CREATED` so the event type should be: 
`# .. event_type: org.openedx.learning.forum.thread.response.created.v1` so it matches